### ISSUE TITLE
The Killer Hues & Scavenge Custom Colors

### ISSUE THREAD

**Issue by MXKhronos** [HelixNebulaStudio/RiseOfTheDead#256]
## The Killer Hues
1. Talk to Lydia and give her a weapon
	- Any hitscan weapon for the primary or secondary slot
2. Look out for zombies breaching your safehome
	- Zombies will start spawning around the safehome.
3. Kill 5 zombies while Lydia watches you kill them
	- Lydia will follow you around and watch you kill zombies
4. Let Lydia kill 10 zombies
	- Let Lydia last hit zombies to count as her kill.
5. Kill the rest of the zombies
6. Talk to Lydia to complete the mission.

Once this mission is complete, accessible Lydia's `NpcWindow` (Safehome > Survivors > Inspect). You will be able to assign her the new task.

## Scavenge Custom Colors
While assigning the task, you'll be require to select a color for Lydia to scavenge based on the color palette picker. Those are the only colors Lydia can scavenge. When this task is active, it will no be available in the assign menu.

Duration: 1h
Can be skipped with perks or gold.

Once task is complete, clicking complete will reward you with the `colorcustom` with the color values of the chosen color. Each stacks up to 5.

**Comment by MrPigZ** [comment]
Scavenged colour will be lost if player clicks complete while inventory is full.

**Comment by MrPigZ** [comment]
this will cause people to miss click
![image](https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/162422862/8b785196-fb3f-44f6-a825-41ff7cf81a72)
![image](https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/162422862/ec0de76e-243a-409c-9d27-4981140f00fa)

**Comment by MrPigZ** [comment]
If player leaves safehome world during mission objective "Kill 5 zombies while lydia watches you kill them" or "Let lydia kill 10 zombies", the mission softlocks

Different safehomes has issues with zombie spawns

Bunker only has 1 working spawnpoint which doesn't spawn zombies out of bounds.

In Slaughter woods has zombie spawning under the ground.
![image](https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/162422862/820ea98d-e155-455e-9cc0-377d9744984f)

Military Camp has zombies spawning under the ground.
![image](https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/162422862/b37b5f6c-9d8b-4e13-a0b0-0fb7144a4964)
![image](https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/162422862/a52913ef-48a4-4f47-a3b3-53c495f85764)

**Comment by Leroy8888** [comment]
Does this mission have any requirements before the player can start it?
